.github/workflows/ci.yml

```diff
@@ -318,11 +318,13 @@ jobs:
 
   complete:
     needs: [webknossos_linux, wkcuber_linux, wkcuber_win, wkcuber_mac, wkcuber_docker, docs]
-    if: |
-      always() &&
-      !contains(needs.*.result, 'failure') &&
-      !contains(needs.*.result, 'cancelled')
+    if: always()
```

Comment:
can this line be removed completely?

Reply:
Nope, unfortunately not, since the default is to skip the step if any predecessors failed or were cancelled.
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-not-requiring-dependent-jobs-to-be-successful

Reply:
ah, good to know. `if: always()` reminds me of `if true`, but I hope that I'll remember this in the future :nerd_face: